java/vector/src/main/java/org/apache/arrow/vector/complex/NullableMapVector.java

```diff
@@ -86,7 +86,8 @@ public NullableMapVector(String name, BufferAllocator allocator, FieldType field
   @Override
   public Field getField() {
     Field f = super.getField();
-    return new Field(f.getName(), true, f.getType(), f.getChildren());
+    FieldType type = new FieldType(true, f.getType(), f.getFieldType().getDictionary(), f.getFieldType().getMetadata());
```

Comment:
oh yeah sorry, I merge that into my working branch - it was actually just merged as a separate PR here: https://github.com/apache/arrow/pull/953
so I think this will end up getting merge-squashed out